hermes-console-vue/src/views/topic/topic-form/TopicForm.vue

```diff
@@ -69,6 +77,15 @@
 
   const showAvroAlert = computed(() => form.value.contentType === 'AVRO');
 
+  const beautify = () => {
+    try {
+      const obj_message = JSON.parse(form.value.schema || '');
+      form.value.schema = JSON.stringify(obj_message, null, 4);
+    } catch (e) {
+      console.warn('Cannot auto format code - incorrect json');
```

Comment:
WDYT about changing console.warn to dispatching notification as we are doing it for example in `SubscriptionForm.vue`?
```
notificationStore.dispatchNotification({
        title: t('notifications.form.validationError'),
        text: '',
        type: 'error',
      });
```